tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -341,18 +364,127 @@ TEST_F(ChargepointTestFixtureV201, K01FR31_IfStartPeriodOfFirstChargingScheduleP
 TEST_F(ChargepointTestFixtureV201, K01FR35_IfChargingSchedulePeriodsAreNotInChonologicalOrder_ThenProfileIsInvalid) {
     auto periods = create_charging_schedule_periods({0, 2, 1});
     auto profile = create_charging_profile(DEFAULT_PROFILE_ID, ChargingProfilePurposeEnum::TxProfile,
-                                           create_charge_schedule(ChargingRateUnitEnum::A, periods), uuid());
+                                           create_charge_schedule(ChargingRateUnitEnum::A, periods), DEFAULT_TX_ID);
 
     auto sut = handler.validate_profile_schedules(profile);
 
     EXPECT_THAT(sut, testing::Eq(ProfileValidationResultEnum::ChargingSchedulePeriodsOutOfOrder));
 }
 
+TEST_F(ChargepointTestFixtureV201, K01_ValidateChargingStationMaxProfile_NotChargingStationMaxProfile_Invalid) {
```

Comment:
we need an FR here or an explanation for why it is not relevant to any one FR

Reply:
This is a simple pre-check for checking that we are passing in the right type of profile. This is not associated with any functional requirement. We think this code is self-documenting.

Reply:
This may be obvious to somebody who has spent the past several months steeped in this code, but it is certainly not self-documenting to somebody who is encountering this code for the first time. Please add a one-line comment that says exactly what you said above "This is a simple pre-check for checking that we are passing in the right type of profile"